CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file, in reverse chronological order by release.
+
+## 0.3.0
+
+### Added
+
+- Return types.
+- Many `InvalidArgumentException`s are thrown when you use invalid arguments. 
+- Integration tests for `UploadedFile` and `ServerRequest`.
+
+### Changed
+
+- We dropped PHP7.0 support. 
+- PSR-17 factories has been marked as internal. They do not fall under or BC promise until PSR-17 is accepted.  
```

Comment:
Should be “under _our_ BC”.
(Is it “factories has been” or “factories have been”? That always gets me.)

Reply:
Thank you